app/reports_generation/scale_profile.yml

```diff
@@ -18,3 +18,4 @@ title: "DCAPT Scale Testing"
 image_height_px: 1000
 image_width_px: 1600
 judge: false
+cold_start: 60
```

Comment:
It would be nice to add a comment, something like:
# skip the first N seconds of the run results as the system is warming up